torch/_higher_order_ops/__init__.py

```diff
@@ -22,4 +39,17 @@
     "PrimHOPBase",
     "foreach_map",
     "_foreach_map",
+    "with_effects",
+    "tag_activation_checkpoint",
+    "auto_functionalized",
+    "auto_functionalized_v2",
+    "associative_scan",
+    "out_dtype",
+    "executorch_call_delegate",
```

Comment:
this seems to shadow the torch._higher_order_ops.executorch_call_delegate module. Is there still a way to import the module?

Reply:
specifically it seems to break this line in ExecuTorch: https://github.com/pytorch/executorch/blob/main/backends/arm/test/runner_utils.py#L172 @zou3519

Reply:
Does `from torch._higher_order_ops.executorch_call_delegate import blah` work? That's how we get around the shadowing in other places here

Reply:
@swolchok in your particular instance we can replace `if isinstance(func, torch._higher_order_ops.executorch_call_delegate.ExecutorchCallDelegate)` with `func is torch.ops.higher_order.executorch_call_delegate` or `func is torch._higher_order_ops.executorch_call_delegate`